WTWSMS/history/characters/WtWSMS_erikw_characters.txt

```diff
@@ -295,7 +295,7 @@
 		birth = yes
 	}
 	421.6.7 = {
-		add_spouse = 1000006
+		add_spouse = 70533
```

Comment:
I don't think this line is needed at all, Theodosius already has:
`421.6.7={
		add_spouse=1000015
	}`

Reply:
Substantially yes, but ErikW always added this line to both spouses.